### ISSUE TITLE
reload() doesn't work when called a second time

### ISSUE THREAD

**Issue by lorriman** [sizzlemctwizzle/GM_config#62]
field['something']].reload(); only works the first time it's called. the original 'wrapper' is removed at the end of the method but nothing replaces it. The code that works for me is ...

``` javascript
GM_configField.prototype['reload']= function() {
    var wrapper = this.wrapper;
    if (wrapper) {
      var fieldParent = wrapper.parentNode;
      newWrapper=fieldParent.insertBefore((this.wrapper = this.toNode()), wrapper);
      this.remove(wrapper);
      this.wrapper=newWrapper;//fix line
    }
  }
```

This was needed needed in the specific case of "select all' and 'deselect all' buttons for a list of checkboxes.

**Comment by sizzlemctwizzle** [comment]
This issue was not resolved. Example that fails to produce expected result:
```js
let gmc = new GM_configStruct(
{
  'id': 'MyConfig', // The id used for this instance of GM_config
  'fields': {
    'Name': // This is the id of the field
    {
      'label': 'Name', // Appears next to field
      'type': 'text', // Makes this setting a text field
      'default': 'Sizzle McTwizzle' // Default value if user doesn't change it
    }
  },
  'events': {
    'open': function() {
      this.fields['Name'].value = 'Mike';
      this.fields['Name'].reload();
      this.fields['Name'].value = 'Medley';
      this.fields['Name'].reload();
    }
  }
});
gmc.open();
```

**Comment by sizzlemctwizzle** [comment]
Closed by #112
